recipe1.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Page</title>
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="functions.js"></script>
+</head>
+
+<body>
+    <div class="navBar">
+        <a href="index.html">Jin's egg-cellent recipes</a>
+        <a href="index.html">Home</a>
+        <a href="about_me.html">About Me</a>
+    </div>
+    <div>
+        <h1 id="name">Boiled Eggs!!!</h1>
+    </div>
+    <p id="json"></p>
```

Comment:
It might be a bit easier to read your JSON if you use the <pre> tag or the <code> tag. (this info was buried in the milestone 2 requirements so no worries if you missed it)